src/components/Breadcrumbs/Breadcrumbs.vue

```diff
@@ -0,0 +1,385 @@
+			if (e.target.contains(e.relatedTarget)) {
+				return
+			}
+			// Get the correct element, in case we leave directly from a child.
+			if (e.target.closest) {
+				const target = e.target.closest('.crumb')
+				if (target.contains(e.relatedTarget)) {
+					return
+				}
+				if (target.classList && target.classList.contains('crumb')) {
+					target.classList.remove('over')
+				}
+			}
+		},
+		openActions() {
+			this.$refs.actions.opened = true
+		}
+	},
+	render: function(createElement) {
+		// Get the breadcrumbs
+		const breadcrumbs = (this.$slots.default || []).filter(node =>
+			!!node
+			&& !!node.componentOptions
+			&& node.componentOptions.tag === 'Breadcrumb'
+		)
+
+		// Check that we have at least one breadcrumb
+		if (breadcrumbs.length === 0) {
+			return
+		}
+
+		// Add the root icon to the first breadcrumb
+		Vue.set(breadcrumbs[0].componentOptions.propsData, 'icon', this.rootIcon)
+
+		const crumbs = []
+		// The first half of the breadcrumbs
+		const crumbs1 = this.hiddenIndices.length
+			? breadcrumbs.slice(0, Math.round(breadcrumbs.length / 2))
+			: breadcrumbs
+		// Todo remove duplicate code
+		crumbs1.forEach((crumb, i) => {
+			if (crumb.elm && crumb.elm.classList) {
+				if (this.isHidden(i)) {
+					crumb.elm.classList.add('hidden')
+				} else {
+					crumb.elm.classList.remove('hidden')
+				}
+			}
+			crumbs.push(crumb)
+		})
+
+		// The Actions menu
+		if (this.hiddenCrumbs.length) {
+			crumbs.push(createElement('div', { class: 'crumb' },
+				[createElement(Actions, {
+					class: 'dropdown',
+					props: {
+						forceMenu: true
+					},
+					ref: 'actions',
+					nativeOn: {
+						dragstart: this.dragstart,
+						dragenter: this.openActions,
+						dragleave: this.closeActions
+					}
+				}, this.hiddenCrumbs.map(crumb => {
+					const path = crumb.componentOptions.propsData.path
+					return createElement(ActionRouter, {
+						class: 'crumb',
+						props: {
+							to: path,
+							icon: 'icon-folder'
+						},
+						attrs: {
+							draggable: false
+						},
+						nativeOn: {
+							dragstart: this.dragstart,
+							drop: ($event) => this.dropped(path, $event),
+							dragover: this.dragOver,
+							dragenter: this.dragEnter,
+							dragleave: this.dragLeave
+						}
+					},
+					crumb.componentOptions.propsData.title
+					)
+				}))]
+			))
+		}
+		// The second half of the breadcrumbs
+		const crumbs2 = this.hiddenCrumbs.length
+			? breadcrumbs.slice(Math.round(breadcrumbs.length / 2))
+			: []
+		crumbs2.forEach((crumb, i) => {
+			if (crumb.elm && crumb.elm.classList) {
+				if (this.isHidden(i + crumbs1.length)) {
+					crumb.elm.classList.add('hidden')
+				} else {
+					crumb.elm.classList.remove('hidden')
+				}
+			}
+			crumbs.push(crumb)
+		})
+
+		return createElement('div', { class: 'breadcrumb', ref: 'container' }, crumbs)
+	}
+}
+</script>
+
+<style lang="scss" scoped>
+@import '../../fonts/scss/iconfont-vue';
+
+.breadcrumb {
+	width: calc(100% - 88px);
```

Comment:
Still don't get this 88px? What does it represent? :)

Reply:
I am not done cleaning up. I will remove it. It is because I had the snapper on the left and an other action on the right, so I needed it to be 88 px smaller than the page width.

Reply:
Sorry, I'll leave this pr alone then xD
Ping be again when you need!

Reply:
No, it's fine. Your comments are very helpful. I just can't completely keep up with them 😅